modules/cluster-logging-visualizer-launch.adoc

```diff
@@ -8,6 +8,23 @@
 {product-title} uses Kibana as the log visualizer. Kibana is a browser-based console to query, discover, and visualize your logs through histograms, line graphs,
 pie charts, heat maps, built-in geospatial support, and other visualizations.
 
+.Prerequisites
```

Comment:
Should we add something like that the "kubeadmin" user does not work. Also, should we mention that usually cluster-admin (and cluster-reader?) is enough?

cc/ @ewolinetz

Reply:
A user must have the `cluster-admin` and `cluster-reader` roles in order to list the *infra* and *audit* indices in Kibana. The default `kubeadmin` user does not have proper permissions to list these indices. 

If you can view the pods and logs in the `default` project, you should be able to access the these indices. You can use the following command to check if the current user has proper permissions:
`$ oc auth can-i get pods/logs -n default
`

Reply:
resolved

Reply:
kubeadmin works now.